knative-operator/pkg/controller/knativekafka/knativekafka_controller.go

```diff
@@ -245,6 +252,10 @@ func (r *ReconcileKnativeKafka) transform(manifest *mf.Manifest, instance *serve
 		}),
 		setKafkaDeployments(instance.Spec.HighAvailability.Replicas),
```

Comment:
Should we track in a Jira the HA setup for the Broker too?

Reply:
yep - mind creating one

Reply:
Done! `SRVKE-1045`